src/NetworkInterface.cpp

```diff
@@ -2461,17 +2476,27 @@ bool NetworkInterface::dissectPacket(u_int32_t bridge_iface_idx,
 	   && !arp_spa_h->isBroadcastDomainHost())
 	  arp_spa_h->setBroadcastDomainHost();
 
+
+  ArpStatsMatrixElement* e;
 	if(arp_opcode == 0x1 /* ARP request */) {
 	  arp_requests++;
 	  srcMac->incSentArpRequests();
 	  dstMac->incRcvdArpRequests();
+
+    e = getArpHashMatrixElement(srcMac->get_mac(), dstMac->get_mac(), true);
+    e->AddOneSentRequests();
```

Comment:
please use the ntopng convention and name this kind of methods similar to incSentArpRequests(), incRcvdArpRequuests() etc..